requirements.txt

```diff
@@ -1,7 +1,7 @@
 anytree==2.8.0
 certifi==2021.5.30
 charset-normalizer==2.0.6
-click==8.0.1
+click==7.1.2
```

Comment:
Why import click? I don't see it used anywhere.

Reply:
Click is a requirement for another package - I believe zipp.

Reply:
Ahh, good reason!